.travis.yml

```diff
@@ -4,12 +4,13 @@ python:
   - "2.7"
   - "3.3"
   - "3.4"
+  - "3.5.0"
```

Comment:
I don't see how that's true. https://travis-ci.org/mitsuhiko/jinja2/builds/81271984